ansible_collection_helper.py

```diff
@@ -7,6 +7,53 @@
 from pathlib import Path
 from typing import Set
 
+import yaml
+from ansible.cli.doc import DocCLI
+from jinja2 import Environment, FileSystemLoader
+
+env = Environment(loader=FileSystemLoader((Path(__file__).parent / "templates")))
+template = env.get_template("readme.jinja2")
+
+
+def generate_role_documentation(
+    role_name: str, role_spec_file: Path, molecule_dir: Path, doc_dir: Path
+) -> None:
+    """Reads roles metadata and molecule converge playbooks to generate README
+    files for a specific role."""
```

Comment:
```suggestion
    files for a specific role.
    """
```
is more common and cleaner because it avoids superfluous diff when later adding extra paragraphs to the docstring.